### ISSUE TITLE
Menu links do not have correct active states

### ISSUE THREAD

**Issue by danieltj27** [Forum-Promotion/help-center#8]
The main navigation menu contains three links at present and at the moment, when you click through each of the three links, they don't always have the correct active `css` class added to them.

When you click on Getting Started, the `navbar__link--active` class is applied correctly. However, clicking on either of the Help Guides of Our Services links applies the `navbar__link--active` class to both the Getting Started link and the respective link you just clicked on.

Long story short, the Getting Started link seems to always have the `navbar__link--active` class applied to it when it shouldn't. I haven't looked into the source code deep enough to figure out a fix so leaving this here in case anyone else can identify the issue.

**Comment by malcolmjr96** [comment]
Thank you for reporting.

This has been fixed in this PR #13
